lib/paypalhttp/version.rb

```diff
@@ -1 +1 @@
-VERSION = "1.0.1"
+VERSION = "1.0.2"
```

Comment:
```suggestion
VERSION = "1.0.1"
```

We'll do this at release time.

Reply:
Are you referring to this step within the release process? 

_gem push paypalhttp-1.X.X.gem_

In the previous deployments we have set the version manually while merging the PR before release: 

- https://github.com/paypal/paypalhttp_ruby/pull/1/files

- https://github.com/paypal/paypalhttp_ruby/pull/11/files

Do we want to approach this the same way or handle the version when we run the release steps?

Reply:
So 2 things that are important here.

First, we shouldn't set the version number until we actually do the release. Doing so within a PR doesn't allow us to do a release with a batch of changes from various PRs.

Second, this is _technically_ a breaking change to change the encoding from `%20` to `+`. It doesn't matter for the PayPal APIs, to my knowledge, but this module is for generic HTTP requests, so it's not guaranteed to be supported in all APIs. (most reasonable API systems will handle both, but we can't be sure about that). So even if we did set the version _in_ this PR, it'd need to be a 2.0.0 version.

Reply:
I completely agree that the version should be 2.0.0 . I will let them know to address these changes.

Well, the reason why we are changing the version number right now is that we plan to make a release right after having this change merged. (To address this ticket: https://engineering.paypalcorp.com/jira/browse/PPPLMER-108542) and so that Cameron can have his change go live before his internship ends. If you advise against making a release I understand, but there aren't typically many changes made to this repo.

Reply:
If you want to handle doing the release as a v2.0.0, then go for it.

Reply:
Ok thanks!